src/private/MDMBlockAnimations.m

```diff
@@ -74,9 +75,9 @@ - (void)addActionForLayer:(CALayer *)layer
 @interface MDMLayerDelegate: NSObject <CALayerDelegate>
 @end
 
-static id<CAAction> ActionForLayer(id self, SEL _cmd, CALayer *layer, NSString *event) {
+static id<CAAction> ActionForKey(CALayer *self, SEL _cmd, NSString *event) {
```

Comment:
Can we keep the parameter named as "layer"? I was confused while reading the body and trying to figure out how `self` was getting referenced from a static function.

Reply:
Good idea - done.